03.bowling/bowling.rb

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+score = ARGV[0]
+
+scores = score.split(',')
+
+shots = []
+scores.each do |s|
+  if s == 'X'
+    shots << 10
+    shots << 0
+  else
+    shots << s.to_i
+  end
+end
+
+frames = shots.each_slice(2).to_a
+
+point = frames.each.with_index.sum do |frame, i|
```

Comment:
```suggestion
point = frames.each.with_index.sum do |current_frame, i|
```

「現在の」を強調してあげると、処理中の`frame`の役割がより明確になりそうです。

Reply:
`each.with_index`を`each_with_index`にするとメソッドの呼び出しが1回減らせますね。

Reply:
`current_frame`にした方がループ内で行われいることの意味が捉えやすいということ、その通りだと思いました🙇‍♂️
また後者のご指摘についても、`each.with_index`で引数を取って添字を変えているわけでもないので、この観点でも`each_with_index`が合理的だと思いました🙇‍♂️修正しております。https://github.com/junohm410/ruby-practices/pull/3/commits/f6ff1b75f8b03f05e7bbe4e9f1d8e92e3edf4ea2